spec/acceptance/php56_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper_acceptance'
+
+describe 'with specific php version' do
+  context 'with params' do
+    it 'works with 5.6' do
+      pp = <<-EOS
+      class { 'php::globals':
+        php_version => '5.6',
+      }
+      -> class { 'php':
+        ensure       => 'present',
+        manage_repos => true,
+        fpm          => true,
+        dev          => true,
+        composer     => true,
+        pear         => true,
+        phpunit      => false,
+      }
+      EOS
+      # Run it twice and test for idempotency
+      apply_manifest(pp, catch_failures: true)
+      apply_manifest(pp, catch_changes: true)
+    end
+
+    case default[:platform]
+    when %r{16.04}
```

Comment:
Isn't this the same as 14.04? Would it be better to test for the Debian OS family or at least Ubuntu?